content/tracing/advanced_usage/_index.md

```diff
@@ -3,6 +3,83 @@ title: Advanced Languages instrumentation
 kind: documentation
 ---
 
+## Trace Analytics Configuration
+
+[Trace Analytics](/tracing/visualization/search/) is used to filter APM Data by [user-defined tags](#custom-tagging) such as `customer_id`, `error_type` or `app_name` to help troubleshoot and filter your requests. Apply the following config to your application to enable this feature.
+
+{{< tabs >}}
+{{% tab "Java" %}}
+
+### Enable Trace Analytics
+
+The following setting will enable trace analytics for all web integrations. (Other types of integrations to be enabled in the future.)
+* System Property: `-Ddd.trace.analytics.enabled=true`
+* Environment Variable: `DD_TRACE_ANALYTICS_ENABLED=true`
+
+### Configure Additional Integrations
+
+To enable or disable Trace Analytics for individual integrations, use the following setting:
+* System Property: `-Ddd.integration.<integration>.analytics.enabled=true`
+* Environment Variable: `DD_INTEGRATION_<INTEGRATION>_ANALYTICS_ENABLED=true`
+
+Integration names can be found on the [integrations table](/tracing/languages/java/#integrations).
+Note: This setting takes priority over the non-integration specific setting.
+
+### Configure Sample Rate
+
+When enabled, the default sample rate is to collect everything (`1.0`). Sampling can be applied with the following config.
+* System Property: `-Ddd.integration.<integration>.analytics.sample_rate=0.5`
+* Environment Variable: `DD_INTEGRATION_<INTEGRATION>_ANALYTICS_SAMPLE_RATE=0.5`
+
+Integration names can be found on the [integrations table](/tracing/languages/java/#integrations).
+
+### Configuring on Custom Instrumentation
+
+Applications with custom instrumentation can enable and set sample rate by setting a tag on the service root span:
+
+```java
+import datadog.trace.api.DDTags;
+import datadog.trace.api.Trace;
+import io.opentracing.Tracer;
+import io.opentracing.util.GlobalTracer;
+
+class SomeClass {
+  @Trace
+  void someMethod() {
+    final Span span = GlobalTracer.get().activeSpan();
+    // Span provided by @Trace annotation.
+    if (span != null) {
+      span.setTag(DDTags.SERVICE_NAME, "my-custom-service");
+      span.setTag(DDTags.ANALYTICS_SAMPLE_RATE, 1.0);
+    }
+  }
+}
+```
+
+{{% /tab %}}
+{{% tab "Python" %}}
+
```

Comment:
I would say to add a message in all missing tabs, such as "Coming Soon. Reach out to the Datadog support team to be part of the beta."

@andrewsouthard1 thoughts?